docs/screenshot-tests.md

```diff
@@ -42,25 +41,58 @@ Option | Description
 `--cap-add=SYS_ADMIN` | Run container with system admin privileges
 `-e MDC_...="${MDC_...}"` | Set MDC_GCLOUD_SERVICE_ACCOUNT_KEY env variable
 
-#### STEP 3: Run Tests
+### STEP 3: Run Tests
 
-At this point you are ready to run. This Docker container is mainly designed for running the screenshot tests. The container already has 1) cloned the repo and 2) `npm install` the project dependencies.
+This Docker container is mainly designed for running the screenshot tests. The container already has 1) cloned the repo and 2) `npm install` the project dependencies.
 
-The next thing you need to do is start the server and run the tests.
+In the Docker container, start the server:
 
 ```
-git checkout <INSERT_YOUR_BRANCH_NAME>
+git pull
+git checkout --track origin/<BRANCH_NAME>
 npm install
 ./test/screenshot/start.sh
 ```
 
 Wait for about 2 - 3 minutes for the server to start up (we know the long wait is an issue). The `start.sh` script is a proxy to `npm start`, but doesn't output to the terminal so you can run other commands.
 
+After waiting 2 - 3 minutes, run the screenshot tests:
+
 ```
 npm run test:image-diff
 ```
 
-This will kick off the screenshot tests. You will see the regular terminal output from mocha.
+You will see the regular terminal output from mocha. Most importantly, it will output two partial URLs for every screenshot test, which you will use to compare screenshots in STEP 4:
+
+```
+<path>/<to>/<file>.html/<commit-hash>/<screenshot-test-id>.diff.png
+<path>/<to>/<file>.html/<commit-hash>/<screenshot-test-id>.snapshot.png
+```
+
+### STEP 4: Debug Screenshot Changes
+
+A `*.snapshot.png` and a `*.diff.png` file will be generated for each screenshot test and uploaded to a Google Cloud Storage bucket. They're publicly accessible through the following URL scheme:
```

Comment:
I didn't know they were public...That was part of the issue https://github.com/material-components/material-components-web-react/issues/122. This PR will close it.